quill-core/src/test/scala/io/getquill/norm/NormalizeNestedStructuresSpec.scala

```diff
@@ -9,7 +9,7 @@ import io.getquill.MirrorContexts.testContext.unquote
 
 class NormalizeNestedStructuresSpec extends Spec {
 
-  val unormalized = quote {
+  val unnormalized = quote {
```

Comment:
I'm not certain about this. Sometimes projects use single letter prefixes for things...